### PR TITLE
Add imath 3.1.9

### PR DESCRIPTION
**Pull request by Vertexwahn**
Add Imath 3.1.9

### CODE REVIEW

File: modules/Imath/3.1.9/patches/add_build_file.patch [hunk 1]
```diff
@@ -0,0 +1,79 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,76 @@
++# SPDX-License-Identifier: BSD-3-Clause
++# Copyright (c) Contributors to the OpenEXR Project.
++
++load("@bazel_skylib//rules:expand_template.bzl", "expand_template")
```
Comment (fmeum) [inline modules/Imath/3.1.9/patches/add_build_file.patch:7]: This needs a skylib `bazel_dep`.

**Comment by Wyverald** [comment]
Note that module names have to be lowercase. So this needs to be named `imath` instead of `Imath` (which is good since I really thought this was called `lmath` at first glance)

**Comment by Wyverald** [comment]
you still need to rename the directory to `imath`, and it looks like one of the patches has a wrong integrity value.

**Comment by Vertexwahn** [comment]
@Wyverald Fix it! 

There is now release archive for Imath

File: modules/imath/3.1.9/patches/add_build_file.patch [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
++        "#cmakedefine IMATH_HALF_USE_LOOKUP_TABLE": "#define IMATH_HALF_USE_LOOKUP_TABLE",
++        "#cmakedefine IMATH_ENABLE_API_VISIBILITY": "#define IMATH_ENABLE_API_VISIBILITY",
++        "#cmakedefine IMATH_HAVE_LARGE_STACK": "/* #undef IMATH_HAVE_LARGE_STACK */",
++        "#cmakedefine01 IMATH_USE_NOEXCEPT": "#define IMATH_USE_NOEXCEPT 1",
++    },
++    template = "config/ImathConfig.h.in",
++)
++
++cc_library(
++    name = "Imath",
```
Comment (fmeum) [inline modules/imath/3.1.9/patches/add_build_file.patch:31]: Could you add an `imath` alias so that `@imath` can be used to reference this dep?

**Review by Wyverald — APPROVED** [review]

**Pull request merged**